nntrainer/models/neuralnet.cpp

```diff
@@ -333,7 +333,16 @@ void NeuralNetwork::save(const std::string &file_path,
       (*iter)->save(model_file);
     }
 
+    model_file.write((char *)&epoch_idx, sizeof(epoch_idx));
+    model_file.write((char *)&iter, sizeof(iter));
```

Comment:
This changes file reading order, which was reading opt type -> epoch idx -> iter number now it's epoch idx -> iter -> opt type -> write opt var(new). this should be updated with where this is being used.